lib/Basic/TaskQueue.cpp

```diff
@@ -85,8 +85,8 @@ bool DummyTaskQueue::execute(TaskQueue::TaskBeganCallback Began,
       std::string Output = "Output placeholder\n";
       std::string Errors =
           P.second->SeparateErrors ? "Error placeholder\n" : "";
-      if (Finished(P.first, 0, Output, Errors, P.second->Context) ==
-          TaskFinishedResponse::StopExecution)
+      if (Finished(P.first, 0, Output, Errors, Optional<TaskFinishedMetrics>(),
```

Comment:
Will `None` work instead of `Optional<TaskFinishedMetrics>`?

Reply:
Yes, `None` is much better. I just learned about `llvm::Optional`, wanted to use a nullptr before 🙈 Btw: is there a map operation on `Optional`?